app/models/federated_protocols/oidc.rb

```diff
@@ -11,15 +11,15 @@ def issuer
     end
 
     def ial
-      request.ial_values.sort.max
+      request.ial_values.first
     end
 
     def aal
-      request.aal_values.sort.max
+      request.aal_values.first
     end
 
     def acr_values
-      [aal, ial].compact.join(' ')
```

Comment:
Leaving this logic as is (although we should report all the values sent, we only store the "highest priority" for a given session")